app/js/App.js

```diff
@@ -1,78 +1,45 @@
 'use strict';
 
-import React              from 'react';
-
-import CurrentUserActions from './actions/CurrentUserActions';
-import CurrentUserStore   from './stores/CurrentUserStore';
-import Header             from './components/Header';
-import Footer             from './components/Footer';
-
-const propTypes = {
-  params: React.PropTypes.object,
-  query: React.PropTypes.object,
-  children: React.PropTypes.oneOfType([
-    React.PropTypes.array,
-    React.PropTypes.object
-  ])
-};
+import React from 'react';
+// import CurrentUserActions from './actions/CurrentUserActions';
+// import CurrentUserStore from './stores/CurrentUserStore';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import IconList from './components/IconList';
+import iconData from './data/iconListInfo';
 
 class App extends React.Component {
 
   constructor(props) {
     super(props);
 
-    this.onUserChange = this.onUserChange.bind(this);
-
     this.state = {
-      currentUser: {}
-    };
-  }
-
-  onUserChange(err, user) {
-    if ( err ) {
-      this.setState({ error: err });
-    } else {
-      this.setState({ currentUser: user || {}, error: null });
+      currentUser: {},
+      icons: iconData
     }
   }
 
+
   componentWillMount() {
     console.log('About to mount App');
   }
 
-  componentDidMount() {
-    this.unsubscribe = CurrentUserStore.listen(this.onUserChange);
-    CurrentUserActions.checkLoginStatus();
-  }
-
-  componentWillUnmount() {
-    this.unsubscribe();
-  }
-
-  renderChildren() {
-    return React.cloneElement(this.props.children, {
-      params: this.props.params,
-      query: this.props.query,
-      currentUser: this.state.currentUser
-    });
-  }
-
   render() {
     return (
+
       <div>
 
         <Header />
 
-        {this.renderChildren()}
+          <IconList icons={this.state.icons} />
```

Comment:
The overall icon list won't change but individual pages will have their own smaller lists attached. I guess what I should do is remove icons from state and pull them in within the IconList object where they're being used for now